core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -1482,9 +1481,9 @@ public synchronized void onBlocksDownloaded(Peer peer, Block block, @Nullable Fi
 
         @Override
         public synchronized void run() {
-            if (blocksInLastSecond > 1) {
```

Comment:
Why should we ever want to suppress those messages? I was heavily confuzed by not seeing them in some cases.